docs/data/material/customization/dark-mode/dark-mode.md

```diff
@@ -342,3 +342,57 @@ For applications that need to support light and dark mode using CSS media `prefe
 
 But if you want to be able to toggle between modes manually, avoiding the flicker requires a combination of CSS variables and the `InitColorSchemeScript` component.
 Check out the [Preventing SSR flicker](/material-ui/customization/css-theme-variables/configuration/#preventing-ssr-flickering) section for more details.
+
+## Storage manager
```

Comment:
Is this the best location for this info? Or would it make sense as an h3 under the System preference header? The fact that it refers back to the Built-in support section for context makes me think they should be located near each other. But if you have a strong opinion about this placement then I'm sure it's fine.

Reply:
I think it should be h2 because it's a different topic similar to `Toggling color mode` and the reason I put this section at the end is because I expect only a few projects looking for this docs.

Do you think it should be moved next to `Toggling color mode`?

Reply:
Got it, in that case yes I think placing it near Toggling color mode makes sense. If you read these two sections back-to-back it does have a logical flow, even if this new addition is somewhat of an edge case.